mapApp/templates/mapApp/edit_hazards.html

```diff
@@ -7,7 +7,7 @@
 {% endblock %}
 
 {% block headerCSS %}
-  <link rel="stylesheet" href="{% static 'leaflet/plugins/awesome-markers/dist/leaflet.awesome-markers.css' %}">
+  <link rel="stylesheet" href="{% static 'leaflet/plugins/extra-markers/css/leaflet.extra-markers.min.css' %}">
```

Comment:
I wonder if we can import the css and js for ExtraMarkers a single time in mapApp/base.html instead of once in each of these four files?

Reply:
I tried out cleaning this up and noticed there are quite a few separate package imports between the home and vis pages... turns out the home page is running leaflet 1.7.1 and the vis page is using 0.7.3, so importing leaflet and their shared packages to base.html doesn't work. I want to redo a bunch of things on the vis page, so I think it makes sense to bookmark this and upgrade vis page leaflet / clean up the packages as part of that feature- does that work for you?

I looked up the 'recent reports' and 'edit hazards' pages for the first time and it doesn't seem to me like either of them are working as-is. Do you know if these are still used, should I look at getting them running again? Can ask Karen & Trisalyn as well.

Reply:
I think your best bet is to ask Karen and Trisalyn about those two pages, I'm not familiar with them. 

I didn't realize we were using two different versions of leaflet! Yes, please feel free to leave things as they are for this PR and consider addressing in the future. Thanks!